Refactoring documentation (#989)
Hi-PACE/hipace: docs/source/run/parameters.rst

General parameters
------------------

* ``hipace.normalized_units`` (`bool`) optional (default `0`)

>     Using normalized units in the simulation.

Reviewer: ```suggestion
    Whether the simulation uses the normalized unit system commonly used in wakefield acceleration, see e.g. `chapter 2 of this reference <https://iopscience.iop.org/article/10.1088/0741-3335/56/8/084012>`__. Otherwise, the code assumes SI (Système International) unit system.
```

* ``random_seed`` (`integer`) optional (default `1`)
    Passes a seed to the AMReX random number generator. This allows for reproducibility of random events such as randomly generated beams, ionization, and collisions.

>     Note that on GPU, since the order of operations is not ensured, the providing of a seed does not guarantee reproducibility to the level of machine precision.

Reviewer: ```suggestion
    Note that on GPU, since the order of operations is not ensured, providing a seed does not guarantee reproducibility to the level of machine precision.
```

* ``hipace.verbose`` (`int`) optional (default `0`)
    Level of verbosity.

      * ``hipace.verbose = 1``, prints only the time steps, which are computed.

      * ``hipace.verbose = 2`` additionally prints the number of iterations in the
        predictor-corrector loop, as well as the B-Field error at each slice.

>       * ``hipace.verbose = 3`` also prints the number of particles, which violate the quasi-static

Reviewer: ```suggestion
      * ``hipace.verbose = 3`` also prints the number of particles that violate the quasi-static
```

    Name of the beam to be read in. If an openPMD file contains multiple beams, the name of the beam
    needs to be specified.

SALAME algorithm
^^^^^^^^^^^^^^^^

> HiPACE++ features the Slicing Advanced Loading Algorithm for Minimizing Energy Spread (SALAME) to generate a beam profile that
> automatically loads the wake optimally, i.e., that the initial wakefield is flattened by the charge of the beam. Important note:

Reviewer: ```suggestion
HiPACE++ features the Slicing Advanced Loading Algorithm for Minimizing Energy Spread (SALAME) to generate a beam profile that
automatically loads the wake optimally, i.e., so that the initial wakefield is flattened by the charge of the beam. Important note:
```

    Background plasma density in SI units. Certain physical modules (collisions, ionization, radiation reactions) depend on the actual background density.
    Hence, in normalized units, they can only be included, if a background plasma density in SI units is provided using this input parameter.

Binary collisions
^^^^^^^^^^^^^^^^^

WARNING: this module is in development.

HiPACE++ proposes an implementation of [Perez et al., Phys. Plasmas 19, 083104 (2012)], inherited from WarpX, between plasma species.
As collisions depend on the physical density, in normalized units `hipace.background_density_SI` must be specified.

* ``plasmas.collisions`` (list of `strings`) optional
    List of names of types binary Coulomb collisions.
    Each will represent collisions between 2 plasma species (potentially the same).

* ``<collision name>.species`` (two `strings`) optional
    The name of the two plasma species for which collisions should be included.

* ``<collision name>.CoulombLog`` (`float`) optional (default `-1.`)
    Coulomb logarithm used for this collision.
    If not specified, the Coulomb logarithm is determined from the temperature in each cell.

> Radiation reactions
> ^^^^^^^^^^^^^^^^^^^
> 
> Whether the energy loss due to classical radiation reactions of beam particles are calculated.
> 

Reviewer: ```suggestion
Radiation reaction
^^^^^^^^^^^^^^^^^^

Whether the energy loss due to classical radiation reaction of beam particles is calculated.

```

* ``<beam name> or beams.do_radiation_reaction`` (`bool`) optional (default `0`)

>     Whether the beam particles undergo energy loss due to classical radiation reactions.

Reviewer: ```suggestion
    Whether the beam particles undergo energy loss due to classical radiation reaction.
```
